src/Sulu/Bundle/MediaBundle/Resources/js/containers/SingleMediaSelection/SingleMediaSelection.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+import React, {Fragment} from 'react';
+import {observer} from 'mobx-react';
+import type {IObservableValue} from 'mobx';
+import {action, autorun, observable, toJS} from 'mobx';
+import SingleItemSelection from 'sulu-admin-bundle/components/SingleItemSelection';
+import {translate} from 'sulu-admin-bundle/utils/Translator';
+import SingleMediaSelectionStore from '../../stores/SingleMediaSelectionStore';
+import MediaSelectionItem from '../../components/MediaSelectionItem';
+import SingleMediaSelectionOverlay from '../SingleMediaSelectionOverlay';
+import type {Value} from './types';
+
+type Props = {|
+    disabled: boolean,
+    locale: IObservableValue<string>,
+    onChange: (selectedIds: Value) => void,
+    value: Value,
+|}
+
+@observer
+export default class SingleMediaSelection extends React.Component<Props> {
+    static defaultProps = {
+        disabled: false,
+        value: { id: undefined },
```

Comment:
We actually do not add padding to the object literals... I thought we had a eslint rule activate for that, but I seem to be wrong... Maybe you can add [this rule](https://eslint.org/docs/rules/object-curly-spacing) in a separate PR?